pyiron_base/jobs/job/core.py

```diff
@@ -776,6 +815,9 @@ def copy_to(
                         posixpath.join(new_job_core.project_hdf5.h5_path, group)
                     ]
             new_job_core._status = "initialized"
+            # THIS PROBABLY DOESN'T WORK! DATABASE NEEDS TO BE UPDATED - SEE KILL METHOD
+            # self.project.db.item_update({"status": "aborted"}, self._job_id)
```

Comment:
At this stage the job is not yet stored in the database, so resenting the internal variable is sufficient.